easybuild/easyconfigs/c/crossguid/crossguid-20190529-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,54 @@
+# Contribution from Imperial College London, UK
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'crossguid'
+local_commit = 'ca1bf4b'
+version = '20190529'
+
+homepage = 'https://github.com/graeme-hill/crossguid'
+description = """
+CrossGuid is a minimal, cross platform, C++ GUID library. 
+It uses the best native GUID/UUID generator on the given platform and has a 
+generic class for parsing, stringifying, and comparing IDs. 
+The guid generation technique is determined by your platform:"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/graeme-hill/crossguid/archive/%s' % local_commit]
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+patches = ['libuuid-20190529.patch']
```

Comment:
can you rename this to `crossguid-20190529_fix_libuuid.patch` or similar, so it's clear this patch belongs to crossguid?

Reply:
Sounds better than what I had initially.